src/fidesops/schemas/user.py

```diff
@@ -0,0 +1,43 @@
+import re
+
+from pydantic import validator
+
+from fidesops.schemas.base_class import BaseSchema
+
+
+class UserCreate(BaseSchema):
+    """Data required to create a FidesopsUser"""
+
+    username: str
+    password: str
```

Comment:
This is sent in the request body, are there benefits to putting this as form data instead, like we do for the `client_id` and `client_secret` in `POST /v1/oauth/token`?